src/traces/parcoords/attributes.js

```diff
@@ -47,6 +49,10 @@ module.exports = {
         }
     },
 
+    labelfont: extendFlat({}, fontAttrs, {dflt: {size: 10}, description: 'Sets the font for the `dimension` labels.'}),
```

Comment:
This is not our usual pattern.

The default font size is inherited from [`fullLayout.font.size`](https://github.com/plotly/plotly.js/blob/460ac39ae99f0e3de46cb9307dc77f581ecbafcb/src/plots/plots.js#L934) directly as [here](https://github.com/plotly/plotly.js/blob/460ac39ae99f0e3de46cb9307dc77f581ecbafcb/src/components/annotations/annotation_defaults.js#L46) or times a scalar like [here](https://github.com/plotly/plotly.js/blob/460ac39ae99f0e3de46cb9307dc77f581ecbafcb/src/plots/cartesian/axis_defaults.js#L83).